install/Knative-with-GKE.md

```diff
@@ -40,16 +40,18 @@ commands will need to be adjusted for use in a Windows environment.
 To simplify the command lines for this walkthrough, we need to define a few
 environment variables.
 
-Set `CLUSTER_NAME` and `CLUSTER_ZONE` variables:
+Set `CLUSTER_NAME` and `CLUSTER_ZONE` variables, you can replace `knative` and
+`us-west1-c` with cluster name and zone of your choosing.
+
+The `CLUSTER_NAME` needs to be lowercase and unique among any other Kubernetes
+clusters in your GCP project. The zone can be
```

Comment:
We shouldn't use acronyms, please use full name for GCP.

Reply:
it has been named GCP across the file before. Do you want me to go and update all of those?

Reply:
Can you add "(GCP)" to the Google Cloud Platform in the before you begin section and use the full name at the beginning of Setting up a Google Cloud Platform project section. Thanks!

Reply:
Done in two places.